docs/admin/multi-cluster.md

```diff
@@ -80,8 +80,9 @@ Second, decide how many clusters should be able to be unavailable at the same ti
 the number that can be unavailable `U`.  If you are not sure, then 1 is a fine choice.
 
 If it is allowable for load-balancing to direct traffic to any region in the event of a cluster failure, then
-you need `R + U` clusters.  If it is not (e.g you want to ensure low latency for all users in the event of a
-cluster failure), then you need to have `R * U` clusters (`U` in each of `R` regions).  In any case, try to put each cluster in a different zone.
+you need at least `min(R, U+1)` clusters.  If it is not (e.g you want to ensure low latency for all users
```

Comment:
I understand your point, and agree. If R is based on serving a certain load then 'R + U' is correct.

But R is not about load, it is to control latency. `To pick the number of clusters, first, decide which regions you need to be in to have adequate latency to all your end users`. To manage load you increase the pods in a cluster, to manage latency you need to be physically close (or go faster than the speed of light).

Obviously, if you have 5 clusters and 4 fail, the remaining cluster needs to be able to scale 500% to maintain availability. This may not be practical. But I think trying to have a single `R` term for both capacity and latency is probably the source of this confusion.

However, I'm not suggesting the recommendations be made more complicated. They just need to provide an example of the process you would use when deciding on the number of clusters. In reality, you would probably decide that you are ok with US-West providing a backup for US-East, but a Europe DC would have too much latency. So it isn't 'either/or' for the two options.